src/reducers/collections.js

```diff
@@ -34,7 +34,8 @@ function validateCollection(configCollection) {
     files,
```

Comment:
This `validateCollection` function is just validating configuration - I'd say move it to `Actions/config` with the rest of the config validation (all of which may be worth porting elsewhere in the future), which allows you to put the new selector in with the rest of the selectors under the `[FOLDER]` key.